e2e/support/helpers/e2e-question-helpers.ts

```diff
@@ -0,0 +1,226 @@
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
```

Comment:
Things in this file have been moved from `e2e/support/commands/api/question.ts`, implementations/interfaces have not been changed.